### DISCUSSION TITLE
Removal of dwaaan/HRConvert2-Docker from README.md

### DISCUSSION THREAD

**Discussion by Presjar** [zelon88/HRConvert2#37] [Ideas]
Seems that the docker image dwaaan was working on is now obsolete and the git was abandoned. 

Is there any way a docker image can be automatically generated on a change to the repo?

**Comment by zelon88** [comment]
This is done.

I also started working on an official docker image and have created an official docker account where I will post the docker image once it is done.

Stay tuned, and thanks to everyone for all their support! The dwaaan Docker repo had a very very good run. It got the project a ton of exposure that it never would have seen without him. I appreciate his work greatly. 

The project really does deserve an officially supported solution, and that's where we are headed.

**Reply by bit-man** [reply]
Please don't forget to add an arm64 image :pray:

**Comment by Presjar** [comment]
That's excellent work!

Well done and thak you.
